src/lib/ngsi/AttributeDomainName.cpp

```diff
@@ -99,11 +99,11 @@ void AttributeDomainName::present(const std::string& indent)
 {
   if (string != "")
   {
-    PRINTF("%sAttributeDomainName: %s\n", indent.c_str(), string.c_str());
+    LM_F(("%sAttributeDomainName: %s", indent.c_str(), string.c_str()));
```

Comment:
DEBUG is used for LM_F.
I don't think there is any protocol to follow for DEBUG messages ...